### DISCUSSION TITLE
Welcome to Spaceblock Discussions!

### DISCUSSION THREAD

**Discussion by IvanIsak2000** [Spaceblock/cosmoterm#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mb6ockatf** [comment]
Hi. I'm gonna collaborate on this project with my fellow @IvanIsak2000

idk what else to write

**Reply by IvanIsak2000** [reply]
This is great! Please tell me what your friend can do

**Reply by mb6ockatf** [reply]
I meant you)) like, you and me are going to work together on this project

**Comment by mb6ockatf** [comment]
Hello, @IvanIsak2000. It's been a while. Finally found some time and will to do something about this project. Probably will commit something this evening.

Should i fork this repository, or commit directly into main?
